### DISCUSSION TITLE
Is there a way in webpack to prevent a function from being dead eliminated forcefully?

### DISCUSSION THREAD

**Discussion by Biboswan** [webpack/webpack#16621] [Q&A]
Usecase: I'm building a chrome extension where 
in `devtools.js` I use
```js
chrome.devtools.inspectedWindow.eval('setSelectedElement($0)', {
        useContentScriptContext: true,
        frameURL: url,
    });
```
And the function `setSelectedElement` in `content` script is called only this way. So bundler is removing this function

.

**Comment by alexander-akait** [comment]
Hello, sorry, I don't undestand, can you clarify?

**Reply by Biboswan** [reply]
I have found a workaround for my problem. 

I wanted to achieve two things
1. the function name ` setSelectedElement` shall be preserved in the bundle.
2. Even if I'm not using this function, it should not be removed from the bundle.

**Reply by alexander-akait** [reply]
webpack doesn't remove such function...
